cli/reporter.py

```diff
@@ -185,6 +185,8 @@ def _collect_assembly_check_results(self):
 
     def create_reports(self):
         report_html = generate_html_report(self.results)
-        with open("report.html", "w") as f:
+        file_path = 'expected_report.html'
```

Comment:
Sorry I didn't explain myself well in the earlier comment - this should still create a file called `report.html`, it's just that the test file used for comparison [here](https://github.com/EBIvariation/eva-sub-cli/pull/2/files/edcfbd0070862d291f770bc73a89e1f39526944d..f0c369393b6dc7ace73566aa31b8960c9c997f1c#diff-59aa45b5e4107d2c022be08538060a2f3701cfe1657eadb979ef9a077cdf8193R57) should be called `expected_report.html`, so it doesn't conflict with the actual report that the code generates, e.g. [here](https://github.com/EBIvariation/eva-sub-cli/pull/2/files/edcfbd0070862d291f770bc73a89e1f39526944d..f0c369393b6dc7ace73566aa31b8960c9c997f1c#diff-42daf86fbfce61647b461b69ee6fee5371e286ed136853ffcc88d2cfafe668b3R26).